src/net_processing.cpp

```diff
@@ -75,6 +75,8 @@ static constexpr std::chrono::microseconds INBOUND_PEER_TX_DELAY{std::chrono::se
 static constexpr std::chrono::microseconds GETDATA_TX_INTERVAL{std::chrono::seconds{60}};
 /** Maximum delay (in microseconds) for transaction requests to avoid biasing some peers over others. */
 static constexpr std::chrono::microseconds MAX_GETDATA_RANDOM_DELAY{std::chrono::seconds{2}};
+/** Delay between receiving a NOTFOUND and trying the next peer. */
```

Comment:
why have any delay?

Reply:
We have some discussion [here](https://github.com/bitcoin/bitcoin/pull/18238#discussion_r396040661)